pkg/cloud/alibaba/assets/config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Delete this file. The config map exists and is created by the installer.

Reply:
ok, I will delete it.